DataFormats/Detectors/ZDC/include/DataFormatsZDC/RecEventFlat.h

```diff
@@ -0,0 +1,180 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef _ZDC_RECEVENTFLAT_H_
+#define _ZDC_RECEVENTFLAT_H_
+
+#include "CommonDataFormat/RangeReference.h"
+#include "CommonDataFormat/InteractionRecord.h"
+#include "DataFormatsZDC/BCRecData.h"
+#include "DataFormatsZDC/ZDCEnergy.h"
+#include "DataFormatsZDC/ZDCTDCData.h"
+#include "ZDCBase/Constants.h"
+#include "MathUtils/Cartesian.h"
+#include <Rtypes.h>
+#include <array>
+#include <vector>
+#include <map>
+
+/// \file RecEventFlat.h
+/// \brief Class to decode the reconstructed ZDC event (single BC with signal in one of detectors)
+/// \author pietro.cortese@cern.ch
+
+namespace o2
+{
+namespace zdc
+{
+using FirstEntry = int;
+using NElem = int;
+
+struct RecEventFlat {
+  o2::InteractionRecord ir;
```

Comment:
The fullCI triggers here a warning `statement should be inside braces`, looks like a false positive.
@davidrohr @timo I don't know who sets up the O2codecheck, do you know?

Reply:
That uses clang-tidy, if there is a bug, there is not much we can do except for waiting for a new clang version
Can you try to put `//NOLING: false positive in clang-tidy`
at the end of line 38, that should silence it. If it doesn't work, I would just force-merge it.

Reply:
Argh, should be `//NOLINT: false positive in clang-tidy` !!

Reply:
thanks, let's see....